client/README.md

```diff
@@ -1,3 +1,23 @@
 ## Client
 
 This is the place for your application front-end files.
+
+Instructions:
+
+1) Make sure the `DB_URL` environment variable is set
+
+2) Type the following from toplevel dir
+
+```
+npm install strongloop
```

Comment:
Ah ok.

In any case you should add it to package.json then and run npm install without the --dev (the dev depedencies are currently just jshint, that is not necessary).